website_analytics_piwik/views/templates.xml

```diff
@@ -1,9 +1,9 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
     <template id="layout" inherit_id="website.layout">
         <xpath expr="//script[@id='tracking_code']" position="after">
-            <t t-if="website and website.piwik_analytics_id and website.piwik_analytics_host">
+            <t t-if="website and website.has_piwik_analytics and website.piwik_analytics_host">
```

Comment:
Maybe also check if the piwik_analytics_id is set, since I as a user can check the bool but not specify the tracking ID. Or just set the piwik_analytics_id (and host) as required when has_piwik_anlytics is true on the config view.